### PR TITLE
fix for issue #15466

### PR DESCRIPTION
**Pull request by RolandCsibrei**
https://github.com/BabylonJS/Babylon.js/issues/15466

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15645/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15645/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15645/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15645/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15645/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15645/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15645/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15645/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15645/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15645/merge/testResults/webgl2playwright/index.html

File: packages/dev/core/src/Misc/tools.ts [hunk 1]
```diff
@@ -364,24 +364,19 @@ export class Tools {
...
         if (
             engine._badDesktopOS &&
             !engine._badOS &&
             // And not ipad pros who claim to be macs...
             !(document && "ontouchend" in document)
         ) {
-            eventPrefix = "mouse";
+            eventPrefix = "pointer";
         }
+        */
```
Comment (RaananW) [inline packages/dev/core/src/Misc/tools.ts:377]: If the fallback is not needed anymore, you can remove the code. we will find it in the history. Maybe leave a single comment saying "a special case was needed for safari, but it was resolved" or something similar

**Comment by RolandCsibrei** [comment]
@RaananW can we please merge this one? I'd like to have the master branch in sync asap. I forgot to put this one into a separate branch. Thank you!!!

**Review by RaananW — APPROVED** [review]
LGTM, waiting for @AmoebaChant to merge.

**Comment by AmoebaChant** [comment]
Thanks for checking with me - I double checked that the [original issue](https://forum.babylonjs.com/t/babylon-js-v4-1-0-beta-24-arcrotatecamera-doesnt-change-view-with-mouse-movement-in-safari-but-works-fine-in-chrome/9710) that this code was introduced to fix no longer repro's on MacOS Safari even with this workaround removed.

**Pull request merged**